distributed/cli/dask_worker.py

```diff
@@ -190,6 +190,13 @@
     show_default=True,
     help="Random amount by which to stagger lifetime values",
 )
+@click.option(
+    "--worker-class",
+    type=str,
+    default="distributed.worker.Worker",
```

Comment:
I have a slight preference to publish this name instead long-term and treat it as the canonical location for these classes.  This is what we use in other parts of the docs as well.

```suggestion
    default="dask.distributed.Worker",
```

The `.worker` bit is purely due to our internal organization of files, and isn't something that users should necessarily see.  I'm happy to guarantee that `from dask.distributed import Worker` will always work, but I'm less sure that this class will always be in the worker.py file.

Reply:
Ah, sorry I misread your original comment